jdbc_bridge/checkstyle.xml

```diff
@@ -0,0 +1,305 @@
+<?xml version="1.0"?>
+<!--
+This file is made available under Elastic License 2.0.
+This file modify from here:
+https://github.com/checkstyle/checkstyle/blob/master/src/main/resources/google_checks.xml
+-->
+
+<!--
+    Checkstyle configuration that checks the Google coding conventions from Google Java Style
+    that can be found at https://google.github.io/styleguide/javaguide.html
```

Comment:
should use one Checkstyle.xml for FE

Reply:
> should use one Checkstyle.xml for FE

fixed by copying config files from Fe during validate stage.